### DISCUSSION TITLE
StackOverflow tag

### DISCUSSION THREAD

**Discussion by paolodina** [collerek/ormar#139] [General]
Well, we have [the first question about ormar on StackOverflow](https://stackoverflow.com/questions/66931694/alembic-migrations-in-ormar-not-working-fastapi). Congrats :tada: ;) I added to the question the tag `ormar`, with the consequence to create a new tag on SO.

I'd like to add some tag description too, I just wonder, do you @collerek agree with something like this? I still have to click on "Save", so please tell me if it's fine for you or what you'd like to be written.

![Screenshot_2021-04-03_16-09-01](https://user-images.githubusercontent.com/1157401/113480965-53b0cb80-9497-11eb-8385-104a15eed1b5.png)

**Comment by collerek** [comment]
Hi,

Thanks for catching that! 😄 

Yep, I guess this is a fine description :)

As for the answer it's caused by using `declarative_base()` from sqlalchemy ORM as a base class which is not allowed, moreover wrong metadata is used in alembic config (the one from Base, not the "raw" one). But since you found the question I will leave the answer for you (unless you want me to answer which is fine, let me know).

And thanks again for catching that :)

**Comment by paolodina** [comment]
Nice. The tag description is now "in" waiting for approval, but I don't see why it shouldn't be. For the answer, nope I won't answer on your behalf, take **your** merits!
